appeals/api/src/server/endpoints/appeals/appeals.controller.js

```diff
@@ -112,10 +112,15 @@ const getMyAppeals = async (req, res) => {
 		);
 		const sortedAppeals = sortAppeals(formattedAppeals);
 
+		// Flatten to an array of strings
```

Comment:
The comment is probably not necessary, but it doesn't hurt

Reply:
No tests fail so it doesn't look like it.  I've also move around a few pages and it looks ok.